R/g.part4.R

```diff
@@ -1083,6 +1083,12 @@ g.part4 = function(datadir = c(), metadatadir = c(), f0 = f0, f1 = f1,
           nightsummary = nightsummary[, which(colnames(nightsummary) %in% c("sleeplatency", "sleepefficiency") ==
                                                 FALSE)]
         }
+        GGIRversion = "GGIR not used"
+        if (is.element('GGIR', installed.packages()[,1])) {
+          GGIRversion = as.character(utils::packageVersion("GGIR"))
+          if (length(GGIRversion) != 1) GGIRversion = sessionInfo()$otherPkgs$GGIR$Version
+        }
+        nightsummary$GGIRversion = GGIRversion
         save(nightsummary, tail_expansion_log, file = paste0(metadatadir, ms4.out, "/", fnames[i]))
```

Comment:
Same comment than in part 2, should the version be stored in a separate object?
`save(nightsummary, tail_expansion_log, GGIRversion, file = paste0(metadatadir, ms4.out, "/", fnames[i]))`